tests/correctness/resources/miniboxing/tests/compile/mb_array_04.check

```diff
@@ -1,3 +1,9 @@
+newSource1.scala:8: warning: Use MbArray instead of Array and benefit from miniboxing specialization
+  val mba1 = MbArray.clone(new Array[Int](10))
+                           ^
+newSource1.scala:9: warning: Use MbArray instead of Array and benefit from miniboxing specialization
+  val mba2 = MbArray.clone(new Array[Double](10))
+                           ^
```

Comment:
Man... I couldn't ask for better students! @Roldak was working yesterday at 3 in the night, you're answering within 1 minute on Sunday at 12pm... What did I do to deserve such cool students? :D :grinning: